core/src/test/java/org/testcontainers/junit/DockerfileTest.java

```diff
@@ -0,0 +1,74 @@
+package org.testcontainers.junit;
+
+import com.github.dockerjava.api.command.BuildImageCmd;
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.pass;
+
+public class DockerfileTest {
```

Comment:
tests for Dockerfile itself will be provided separately as Unit tests with string-based comparation